### ISSUE TITLE
[Embeddable tool] Header bar text color

### ISSUE THREAD

**Issue by mollieru** [votinginfoproject/gttp-relaunch#124]
I might have missed the decision to switch the background color of the top bar on the embeddable tool. If this was an intentional call, I'd just say we need to update the text color for accessibility purposes -- I suggest making the 'Your Voting Access' and the hamburger menu icon white instead of the maroon. 

Currently:
<img width="1197" alt="screen shot 2018-08-07 at 1 32 08 pm" src="https://user-images.githubusercontent.com/3680494/43792422-966aa3a4-9a46-11e8-8393-749dee01ba54.png">

Suggestion: 
<img width="1201" alt="screen shot 2018-08-07 at 1 36 40 pm" src="https://user-images.githubusercontent.com/3680494/43792532-f3dbef84-9a46-11e8-8a25-7f5ee76534f5.png">

**Comment by MariaBianchi** [comment]
I don't believe this was an intentional choice so much as it was a function of the way this got implemented, but thanks for flagging the accessibility issue - Let me see if I can unwind this.
